src/transformers/models/longformer/modeling_tf_longformer.py

```diff
@@ -1758,12 +1762,18 @@ def _pad_to_window_size(
         batch_size, seq_len = input_shape[:2]
         padding_len = (attention_window - seq_len % attention_window) % attention_window
 
-        if padding_len > 0:
+        def log():
             logger.info(
                 "Input ids are automatically padded from {} to {} to be a multiple of `config.attention_window`: {}".format(
                     seq_len, seq_len + padding_len, attention_window
                 )
             )
+            return 0
+
+        def not_log():
+            return 1
+
+        _ = tf.cond(tf.math.greater(padding_len, 0), log, not_log)
```

Comment:
Really don't want to have this as well. I don't want to have "fake functions" in TF - I'm sure there are better ways. This worked before no? Why do we have to change it now exactly?